### DISCUSSION TITLE
LDAP  access denied: user 'xxxx' - Not management user

### DISCUSSION THREAD

**Discussion by raynent** [docker-library/rabbitmq#689] [Q&A] [answered]
docker compose file 
![image](https://github.com/docker-library/rabbitmq/assets/143066456/22d10caf-5440-4f1f-b3e4-6f85fa2cf688)

LDAP config
![image](https://github.com/docker-library/rabbitmq/assets/143066456/f710e13e-980a-4910-9e23-eeee5a9fddb0)

ERROR showing when i login with ldap account
![image](https://github.com/docker-library/rabbitmq/assets/143066456/f3dfb0d5-5330-4c99-adf2-40043d576164)
![image](https://github.com/docker-library/rabbitmq/assets/143066456/8533e34f-d612-42ed-a8a4-c6c7aaddfb63)

Any idea ? thanks a lot

**Comment by lukebakken** [comment, marked as answer]
Hello, thanks for using RabbitMQ and this docker image.

In general, if a GitHub repository has "Discussions", you should use that feature unless you are **absolutely certain** you have found a bug.

In addition, The [`docker-library/rabbitmq`](https://github.com/docker-library/rabbitmq) repository is _not_ the correct place to report this issue. You should start a discussion here:

https://github.com/rabbitmq/rabbitmq-server/discussions

Once you start a discussion there, I will answer your question.

**Reply by raynent** [reply]
sure thanks
